bsyncviewer/lib/bedes/v2.5/schema2.4.0/bedes-mappings-enumerations.csv

```diff
@@ -953,10 +953,10 @@ therms/h,40109,0534d11a-ec7b-47fb-a23a-c4739ae0ca79,therms/hr,therms per hour,ht
 lbs/h,40110,27e7f542-5e15-498b-b033-32e08daa2a4f,lbs/hr,pounds per hour,https://bedes.lbl.gov/node/27e7f542-5e15-498b-b033-32e08daa2a4f,10645e3b-270a-4251-8153-36017540d539,0.966667,49215
 Klbs/h,40111,36cd0e92-79fb-4fbf-a18f-5283869e55f8,klbs/hr,thousand pounds per hour,https://bedes.lbl.gov/node/36cd0e92-79fb-4fbf-a18f-5283869e55f8,10645e3b-270a-4251-8153-36017540d539,0.99,49215
 Mlbs/h,40112,ce93df08-b83e-4c25-8d2e-2c1a2655ff5f,Mlbs/hr,million British thermal units per hour,https://bedes.lbl.gov/node/ce93df08-b83e-4c25-8d2e-2c1a2655ff5f,10645e3b-270a-4251-8153-36017540d539,0.971429,49215
-Cooling ton,40113,437f592b-2605-4be4-bffc-debf6de305d4,cooling tons,cooling tons,https://bedes.lbl.gov/node/ce93df08-b83e-4c25-8d2e-2c1a2655ff5ff,10645e3b-270a-4251-8153-36017540d539,0.99,49215
-Other,40114,1bf118dd-9a9e-440c-8e8f-b3a92ab62855,Other,Other,https://bedes.lbl.gov/node/1bf118dd-9a9e-440c-8e8f-b3a92ab62855,1d87e8ff-747e-4a30-8342-e43b96e53bd4,1,49215
-Roof,40115,fe6a5ef0-9365-4932-bed7-eb0ca9df5c0b,Roof,Makes up theÂ top exterior boundary of the premises envelope.Â It isÂ generally horizontal or sloped less than 60 degrees from horizontal. Represents the complete roof construction.,https://bedes.lbl.gov/node/fe6a5ef0-9365-4932-bed7-eb0ca9df5c0b,686035b9-341d-490b-ad08-b5c192b701b1,1,49222
-Mechanical Room,40116,cd0e8b58-59bb-4e6c-b153-301be7e32785,Mechanical room,The mechanical room is dedicated to the mechanical equipment and its associated electrical equipment.,https://bedes.lbl.gov/node/cd0e8b58-59bb-4e6c-b153-301be7e32785,c96589bd-0dfc-47a4-b040-11dba49d2ebd,0.973333,49222
+Cooling ton,40113,437f592b-2605-4be4-bffc-debf6de305d4,cooling tons,cooling tons,https://bedes.lbl.gov/node/437f592b-2605-4be4-bffc-debf6de305d4,10645e3b-270a-4251-8153-36017540d539,0.99,49215
+Other,40114,1bf118dd-9a9e-440c-8e8f-b3a92ab62855,Other,Other,https://bedes.lbl.gov/node/437f592b-2605-4be4-bffc-debf6de305d4,1d87e8ff-747e-4a30-8342-e43b96e53bd4,1,49215

```

Comment:
I must have made a selection mistakes. The lines after `Cooling ton` should not be changed.

Reply:
doh, can you create a PR with the fix then.

Reply:
#159